packages/teams-js/test/public/media.spec.ts

```diff
@@ -838,7 +713,146 @@ describe('media', () => {
           },
         } as DOMMessageEvent);
 
-        return expect(promise).rejects.toEqual({ errorCode: ErrorCode.SIZE_EXCEEDED });
+        await expect(promise).rejects.toMatchObject({ errorCode: ErrorCode.SIZE_EXCEEDED });
+      });
+    });
+  });
+
+  describe('videoController', () => {
+    describe('v1', () => {
+      it('videoController notifyEventToHost is handled successfully', async () => {
+        expect.assertions(7); // initializeWithContext has 3 assertions + 4 in this test = 7
+
+        await mobilePlatformMock.initializeWithContext(FrameContexts.task, HostClientType.android);
+        mobilePlatformMock.setClientSupportedSDKVersion(nonFullScreenVideoModeAPISupportVersion);
+        const sendMessageToParentSpy = jest.spyOn(communication, 'sendMessageToParent');
+
+        await expect(new media.VideoController().stop(emptyCallback)).resolves.not.toThrow();
+
+        const message = mobilePlatformMock.findMessageByFunc('media.controller');
+        expect(message).not.toBeNull();
+        expect(message.args.length).toBe(1);
+        const callbackId = message.id;
+
+        mobilePlatformMock.respondToMessage({
+          data: {
+            id: callbackId,
+            args: [undefined],
+          },
+        } as DOMMessageEvent);
+
+        expect(sendMessageToParentSpy).toHaveBeenCalled();
+      });
+
+      it('videoController notifyEventToHost is not handled successfully', async () => {
```

Comment:
A slightly more descriptive name will help people understand what test is failing if it ever does. Maybe something like "stop returns SdkError to callback when parent rejects message"?